evaluate/task/task-transpile.go

```diff
@@ -191,3 +187,44 @@ func (t *TaskTranspile) unpackTranspilerPackage(ctx evaltask.Context, fileLogger
 
 	return originFilePath, stubFilePath, nil
 }
+
+// validateTranspileRepository checks if the repository for the "transpile" task is well-formed.
+func validateTranspileRepository(logger *log.Logger, repositoryPath string, destinationLanguage language.Language) (err error) {
+	logger.Printf("validating repository %q", repositoryPath)
+
+	var originLanguage language.Language
+	if _, ok := destinationLanguage.(*golang.Language); ok {
+		originLanguage = &java.Language{}
+	} else {
+		originLanguage = &golang.Language{}
+	}
```

Comment:
Can you please already open the Ruby support issue and note that these switches need to be adapted (we do this somewhere else as well AFAIK)?

Reply:
Ah the ruby support issue exists already. Will put it there.

Reply:
We also do it in the `Run` method.

Reply:
Added to https://github.com/symflower/eval-dev-quality/issues/300